source/boards/accessing-boards.rst

```diff
@@ -18,7 +18,7 @@ Access your Boards
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
-You can access all your boards as well as boards in channels you belong to in the Boards dashboard. Select **Boards** from the Product menu in the top-left corner of Mattermost to open the dashboard and switcher. You can also select the Boards icon in the channel header to open the board for the current channel, Direct Message, or Group Message.
+You can access all your boards as well as boards in channels you belong to in the Boards dashboard. Select **Boards** from the Product menu in the top-left corner of Mattermost to open the dashboard. You can also select the Boards icon in the channel header to open the board for the current channel, Direct Message, or Group Message.
```

Comment:
Just a note: This is actually called the switcher in-product which is why it's in here post 6.0. :) 

![Screen Shot 2021-11-08 at 20 40 14](https://user-images.githubusercontent.com/4223530/140798987-674bb308-c717-4c79-afd3-6a6ab163000c.png)

Reply:
Thanks! I'll revert this change. I suspect that this in-product term will likely need to change in the future given similar terminology changes we're in the process of making.

Reply:
Agreed! I'll discuss it with the Focalboard team.

Reply:
Raised here: https://community.mattermost.com/core/pl/j5stsoe43pymzcxjrgjpmjqbko

Reply:
As per the thread, I'll make this update but it will shortly be moot. :)